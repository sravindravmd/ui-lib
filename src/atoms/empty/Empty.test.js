import React from 'react'
import { Empty } from '.'
import { render, cleanup } from '@testing-library/react'
import 'regenerator-runtime/runtime'

afterEach(cleanup)
const args = {
  className: 'empty',
  primaryText: 'You have no master hierarchy',
  secondaryText: 'Start by adding your master h’s and add\nany published h’s',
  buttonProps: {
    text: 'Import Master hierarchy',
    size: 'small',
  },
}
describe('<Empty/>', () => {
  it('renders empty placeholder', () => {
    const { container } = render(<Empty {...args} />)
    expect(container).toMatchSnapshot()
  })
})
