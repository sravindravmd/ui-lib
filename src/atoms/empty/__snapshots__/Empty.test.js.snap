// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Empty/> renders empty placeholder 1`] = `
<div>
  <div
    class="sc-bdfBwQ hxWtuT empty"
  >
    <div
      class="empty-inner"
    >
      <div
        class="primary-text"
      >
        You have no master hierarchy
      </div>
      <div
        class="secondary-text"
      >
        Start by adding your master h’s and add
any published h’s
      </div>
      <div
        class="button-block"
      >
        <button
          class="sc-gsTCUz gTkrDu"
        >
          Import Master hierarchy
        </button>
      </div>
    </div>
  </div>
</div>
`;
